app/webpacker/src/stylesheets/_commodity-tree.scss

```diff
@@ -186,6 +186,24 @@ article.tariff {
       position:relative;
       word-wrap: break-word;
 
+    // Creates the hiphen lines for the commodities 
+      @for $i from 2 through 50 {
+        &.level-#{$i} {
+            $url_list: ();
+            $position_list: ();
+            $dashes_spacing: 20px;
```

Comment:
When we discussed this we concluded, `$dashes_spacing` must always be the same value as `$token_list`

Please either chain the variable from `$token_width` (ie `$dashes_spacing: $token-width`) or drop it entirely and just use the `$token_width` variable instead on line 198. Thanks

Reply:
thanks for the reminder, I'll go with
`$dashes_spacing: $token-width`

I think it makes clearer the purpose of the variables (even if the value is the same)